rrmngmnt/network.py

```diff
@@ -583,3 +588,55 @@ def if_down(self, nic):
         cmd = "ip link set down %s" % nic
         rc, _, _ = self.host.run_command(shlex.split(cmd))
         return not bool(rc)
+
+    def is_ssh_connective(self, tcp_timeout=20.0):
```

Comment:
sorry I meant under ssh module and call it is_connective, without 'ssh' because it is already under ssh module.
here should be is_connective method as well, but with different check: that host is available over network. ping for example ..